### PR TITLE
fix: add some missing English translations

### PR DESCRIPTION
**Pull request by phlpsong**
missing English translations in the Menu and General tab.

### CODE REVIEW

**Review by tisfeng — CHANGES_REQUESTED** [review]

File: Easydict/App/Localizable.xcstrings [hunk 1]
```diff
@@ -560,9 +560,15 @@
...
+        "en" : {
+          "stringUnit" : {
+            "state" : "needs_review",
+            "value" : "Check for Updates"
+          }
+        },
         "zh-Hans" : {
           "stringUnit" : {
-            "state" : "translated",
+            "state" : "needs_review",
```
[the remark this answers is not in the document]
Comment (phlpsong) [reply Easydict/App/Localizable.xcstrings:571]: OK, I thought it needed to be marked as this, changed.

**Comment by tisfeng** [comment]
Please check all Localizable.xcstrings, including Chinese and English, mark them all as reviewed, some of them may not be used, just ignore them.

image

**Review by tisfeng — APPROVED** [review]

**Review by Jerry23011 — APPROVED** [review]

File: Easydict/App/Localizable.xcstrings [hunk 2]
```diff
@@ -2639,9 +2651,15 @@
     },
     "setting.general.voice.auto_play_word_audio" : {
       "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "needs_review",
+            "value" : "Auto play pronunciation after querying English words"
```
[the remark this answers is not in the document]
Comment (Jerry23011) [reply Easydict/App/Localizable.xcstrings:2676]: leaving it here for later adjustments

**Comment by phlpsong** [comment]
@tisfeng @Jerry23011 If there are no other issues, please kindly merge this.

**Comment by tisfeng** [comment]
ok

**Pull request merged**